cmd/modulectl/create/module/flags.go

```diff
@@ -0,0 +1,60 @@
+package module
+
+import (
+	"github.com/spf13/pflag"
+
+	"github.com/kyma-project/modulectl/internal/service/module"
+)
+
+const (
+	// TODO: Can be extracted to one higher level
```

Comment:
I don't see the value of this `TODO`, which higher level?

Reply:
[Here](https://github.com/kyma-project/modulectl/pull/40#issuecomment-2312511452) I gave a brief explanation of its purpose. But I agree it should be handled separately. Removing the TODOs.